src/java/htsjdk/samtools/util/SimpleInterval.java

```diff
@@ -0,0 +1,45 @@
+package htsjdk.samtools.util;
+
+
+/**
+ * minimal immutable class representing a 1-based closed ended genomic interval
+ */
+public class SimpleInterval implements Locatable {
+
+    private final int start;
+    private final int end;
+    private final String contig;
+
+    public SimpleInterval(final String contig, final int start, final int end){
+        if(contig == null){
+            throw new IllegalArgumentException("contig cannot be null");
```

Comment:
It's not an absence of location, it's a location not in the reference.  ```Locatable``` is relative to the reference implicitly, so you can say that it is "not in the reference".  I would not use "unmapped" here since that is confusing to a process (i.e. mapping).  I would ```inLocation``` or ```inReference``` or some positive term (I don't like these terms).